### PR TITLE
Validation test has been implemented

### PR DESCRIPTION
**Pull request by A9afon**
Validation test has been implemented

### CODE REVIEW

File: src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java [hunk 1]
```diff
@@ -157,6 +158,65 @@ void shouldSaveRadarDto() {
     Mockito.verify(radarTypeRepository).findById(radarType.getId());
   }
 
+  @Test
+  void shouldSaveAnyPrimaryRadarDto() throws Exception {
```
Comment (arufanov) [inline src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java:162]: @A9afon 
* I don't understand the idea.

File: src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java [hunk 2]
```diff
@@ -157,6 +158,65 @@ void shouldSaveRadarDto() {
...
+    Mockito.when(radarTypeRepository.findById(any())).thenReturn(Optional.of(radarType));
+
+    InvalidPrimaryException exception =
+        catchThrowableOfType(() -> radarService.save(radarMapper.toDto(radar)),
+            InvalidPrimaryException.class);
+    Assertions.assertFalse(exception.getMessage().isEmpty());
+  }
+
+  @Test
+  void shouldSaveAnyPrimaryRadarDtoWithAnyRadar() throws Exception {
```
Comment (arufanov) [inline src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java:185]: @A9afon 
* I don't understand the idea.

File: src/test/java/com/a5lab/axion/domain/radar/RadarCfgControllerTests.java [hunk 3]
```diff
@@ -212,6 +211,66 @@ public void shouldFailToCreateRadarDueToNotUniqueTitle() throws Exception {
...
+            .contentType(MediaType.APPLICATION_FORM_URLENCODED)
+            .param("radarType.id", String.valueOf(radarDto.getRadarTypeId()))
+            .param("title", radarDto.getTitle())
+            .param("description", radarDto.getDescription())
+            .sessionAttr("radarDto", radarDto))
+        .andExpect(status().isOk())
+        .andExpect(view().name("settings/radars/add"))
+        .andReturn();
+
+    String content = result.getResponse().getContentAsString();
```
Comment (arufanov) [inline src/test/java/com/a5lab/axion/domain/radar/RadarCfgControllerTests.java:268]: @A9afon 
* see https://github.com/a5lab/axion/issues/657 can we use attributeHasFieldErrorCode?

**Review by arufanov — CHANGES_REQUESTED** [review]

File: src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java (hunk 1 above)
Comment (A9afon) [reply src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java:162]: @arufanov 
I didn't see it, my mistake

File: src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java (hunk 2 above)
Comment (A9afon) [reply src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java:184]: @arufanov
I didn't see it, my mistake

**Review by arufanov — APPROVED** [review]

**Pull request merged**